docs/examples/Ch-ExamplesAddingModbusDevice.rst

```diff
@@ -2,459 +2,339 @@
+Path defines the Modbus device's unit ID (or slave ID).
 
-In this example the id is “5a1dd585e4b0c3936013123d”
+In the RTU protocol, address is defined in five comma-separated parts:
 
-Next you want to get the “name” of the command you want to schedule an event for
+* serial port
+* baud rate 
+* data bits
+* stop bits
+* parity (N - None is 0, O - Odd is 1, E - Even is 2, default is E).
 
 ::
 
-  "commands": [
-              {
-                 "id": "5a1dcdfce4b0c39360131239",
-                 "name": "TemperatureDegF", <--- This
-                 "get": {
-                     "url": "http://localhost:48082/api/v1/device/5a1dd585e4b0c3936013123d/command/5a1dcdfce4b0c39360131239",
-                     "responses": [
-                         {
-                             "code": "200",
-                             "description": "Get the temperature in degrees F",
-                             "expectedValues": [
-                                 "TemperatureDegF"
-                             ]
-
+    [Logging]
+    EnableRemote = false
+    File = "./device-Modbus.log"
+    Level = "DEBUG"
+
+    [Device]
+      DataTransform = true
+      InitCmd = ""
+      InitCmdArgs = ""
+      MaxCmdOps = 128
+      MaxCmdValueLen = 256
+      RemoveCmd = ""
+      RemoveCmdArgs = ""
+      ProfileDir = "/custom-config"
+
+    # Pre-define Devices
+    [[DeviceList]]
+      Name = "Modbus TCP test device"
+      Profile = "Network Power Meter"
+      Description = "This device is a product for monitoring and controlling digital inputs and outputs over a LAN."
+      labels = [ "Air conditioner","modbus TCP" ]
+      [DeviceList.Addressable]
+        name = "Gateway address 1"
+        Protocol = "TCP"
+        Address = "10.211.55.6"
+        Port = 502
+        Path = "1"
+    
+    [[DeviceList]]
+      Name = "Modbus TCP test device 2"
+      Profile = "Network Power Meter"
+      Description = "This device is a product for monitoring and controlling digital inputs and outputs over a LAN."
+      labels = [ "Air conditioner","modbus TCP" ]
+      [DeviceList.Addressable]
+        name = "Gateway address 1"
+        Protocol = "TCP"
+        Address = "10.211.55.6"
+        Port = 502
+        Path = "2"
+
+   # Pre-define Schedule Configuration
+    [[Schedules]]
+    Name = "20sec-schedule"
+    Frequency = "PT20S"
+
+    [[ScheduleEvents]]
+    Name = "Read Switch status"
+    Schedule = "20sec-schedule"
+      [ScheduleEvents.Addressable]
+      HTTPMethod = "GET"
+    Path = "/api/v1/device/name/Modbus TCP test device 1/Configuration"
+
+    [[ScheduleEvents]]
+    Name = "Put Configuration"
+    Parameters = "[{\"DemandWindowSize\": \"110\"},{\"LineFrequency\": \"50\"}]"
+    Schedule = "20sec-schedule"
+      [ScheduleEvents.Addressable]
+      HTTPMethod = "Put"
+      Path = "/api/v1/device/name/Modbus TCP test device 1/Configuration"
+
+You can download and use the provided :download:`configuration.toml
```

Comment:
You will need to upload this configuration.toml file to this directory, and give it a unique name like EdgeX_ExampleModbus_configuration.toml

Reply:
Files added to directory and where necessary link text updated